hysds/job_worker.py

```diff
@@ -1438,6 +1438,26 @@ def handler(signum, frame):
             "celery_hostname": run_job.request.hostname,
         }
 
+    # overwrite message if _alt_msg.txt was dumped
+    msg = list()
+    alt_msg_file = os.path.join(job_dir, "_alt_msg.txt")
+    if os.path.exists(alt_msg_file):
+        with open(alt_msg_file) as f:
+            msgs = f.readlines()
+            logger.info("Got alternate info message: %s" % msgs)
+        for m in msgs:
```

Comment:
should the `for` loop be in the `with` block?

Reply:
cool. thanks. done.